### PR TITLE
Development

### PR DESCRIPTION
**Pull request by AlejoCode**
Please check my methods to see if they are done correctly I will be waiting any response for you guys thank you so much and have a great day :D

### CODE REVIEW

**Comment by bellom** [comment]
## Currently reviewing this PR

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @AlejoCode, 

Great work, all methods working just few things to do and have your :+1: .

To consider the project 'complete' it must satisfy the following Microverse guidelines key points; 
*  [ ]  The project has a descriptive README.md
*  [ ]  Make sure that code is clean: indentation with two spaces, line lengths, spaces around operators - first section of [this link](https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide).

Kindly correct these few things and submit another code review.....

**Comment by bellom** [comment]
## Currently reviewing this PR

For some reason, you are yet to make the correction i mentioned in my previous review.
1. kindly add README.
2. fix the remaining spacing around operators.
I sent you a message on slack but no response, i wish you could fix the error now, and move on to the next project.

File: enumerable.rb [hunk 1]
```diff
@@ -0,0 +1,118 @@
+module Enumerable
+  def my_each
+      i = 0
+    while i < self.size
+      yield(self[i])
+      i+=1
```
Comment (bellom) [inline enumerable.rb]: space should be around operator

File: enumerable.rb [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    end
+  end
+
+  def my_each_with_index
+    i = 0
+    while i < self.size
+      yield(self[i], i)
+      i+=1
```
Comment (bellom) [inline enumerable.rb]: space should be around operator

File: enumerable.rb [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+      end
+    else
+      self.my_each do |e|
+        count += 1 if yield e
+      end
+    end
+    count 
+  end
+
+  def my_map proc=nil
```
Comment (bellom) [inline enumerable.rb]: space should be around operator

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @AlejoCode, 
#### Waited for more than 30 mins hoping to get a response from you.
_Free to messages me on slack if you need help or anything you don't understand._
**_kindly make the changes i mentioned in my first review and re-submit another code review._**

**Comment by AdnanAfsari** [comment]
Adnan Afsari working on it :)

**Comment by AdnanAfsari** [comment]
Hello @AlejoCode 

Your project should have README.md and it should be well documented to get approval :)

Please submit for another review after making the change.

Happy Coding

**Comment by JamesOkunlade** [comment]
James working on it.

**Review by JamesOkunlade — APPROVED** [review]
You have added an updated README.md as requested by the previous reviewer.

It's a :+1: from me. Merge to master, close pull request and fill the project completion form.

Good luck with your future projects.

**Pull request merged**